plugins/android/src/main/java/tools/forma/android/utils/android.kt

```diff
@@ -40,7 +36,49 @@ internal fun CompileOptions.applyFrom(config: FormaConfiguration) {
 }
 
 internal fun NamedDomainObjectContainer<BuildType>.applyFrom(config: BuildConfiguration) {
-    config.buildTypes.forEach { (name, lambda) ->
-        lambda(getByName(name))
+    when (config) {
+        is Debug -> getByName("debug") {
```

Comment:
Are you sure that we have to use single binary for configuration? We still can build task release-binary:assembleDebug or using release-binary:assemble actually builds both release and debug warrant